nautobot/extras/models/datasources.py

```diff
@@ -65,6 +67,23 @@ class GitRepository(PrimaryModel):
         default="",
     )
 
+    username_secret = models.ForeignKey(
+        to=Secret,
```

Comment:
Instead of introducing another import inter-dependency I feel it would be better to use the string form `"extras.Secret"` here and eliminate the import of the `Secret` model.

Reply:
Good point, will do.